code/controllers/Processes/shuttles/supply.dm

```diff
@@ -493,13 +493,18 @@
 			return 1
 		var/crates = 1
 		if(multi)
-			var/num_input = input(usr, "Amount:", "How many crates?", "") as num
-			crates = Clamp(round(text2num(num_input)), 1, 20)
+			var/num_input = input(usr, "Amount:", "How many crates?") as null|num
+			if(!num_input)
+				return 1
+			crates = Clamp(round(num_input), 1, 20)
 			if(..())
```

Comment:
You can probably combine the !num_input check and the ..() check. The ..() check is in place to prevent people from walking away from the console with the input menu open.